tests/session/session-debug-005.phpt

```diff
@@ -68,6 +69,12 @@ object(MongoDB\Driver\Session)#%d (%d) {
   ["transactionState"]=>
   string(11) "in_progress"
   ["transactionOptions"]=>
-  NULL
+  array(1) {
+    ["readPreference"]=>
+    object(MongoDB\Driver\ReadPreference)#%d (%d) {
+      ["mode"]=>
+      string(7) "primary"
+    }
+  }
```

Comment:
FYI: Even though the test doesn't specify a primary read preference, this is ultimately inherited from the Manager itself (where "primary" is the default). The rules for this are discussed in the [Transactions spec](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#readpreference).